.lintstagedrc.mjs

```diff
@@ -51,7 +51,10 @@ export default {
     ], {});
 
     const mrlCommands = mrlFiles.length > 0
-      ? ["monorepolint check --verbose", `dprint fmt ${mrlFiles.join(" ")}`]
+      ? [
+        "monorepolint check --verbose",
+        `dprint fmt ${mrlFiles.join(" ")} --allow-no-files`,
```

Comment:
Why did we have to add this here?

Reply:
Because there is an edge case where we end up passing files to dprint fmt that are in the exclude list and dprint removes those files from "its list to format" resulting in an empty list and then an error.